src/App.scss

```diff
@@ -1 +1,76 @@
-// styles go here
```

Comment:
I would recommend splitting rules into small groups according to their purpose (e.g. positioning, formatting)